SourceSVN/lang/strings_english.txt

```diff
@@ -24,5 +24,5 @@ $s_plugin_SourceSVN_svnssl = 'SVN: Trust All SSL Certs<br/><span class="small">(
 $s_plugin_SourceSVN_winstart = 'SVN: Use Windows `start`<br/><span class="small">(Requires a configured path to binary)</span>';
 
 $s_plugin_SourceSVN_error_path_invalid = 'Path to Subversion binary is invalid, inaccessible or not a directory.';
-$s_plugin_SourceSVN_error_run_svn = 'shell_exec() failed to execute Subversion.';
-
+$s_plugin_SourceSVN_error_svn_run = 'Failed to execute Subversion.  Command returned "%1$s"';
+$s_plugin_SourceSVN_error_svn_cmd = 'svn execution error: "%1$s".';
```

Comment:
The error messages are very similar - is it really worth having 2 different strings ?